flake.nix

```diff
@@ -0,0 +1,111 @@
+{
+  description = "🎁 generate beautiful landing pages for your developer tools";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+    fenix = {
+      url = "github:nix-community/fenix";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+  };
+
+  outputs =
+    { self
+    , nixpkgs
+    , flake-utils
+    , fenix
+    , ...
+    }:
+    flake-utils.lib.eachDefaultSystem
+      (system:
+      let
+        pkgs = import nixpkgs {
+          inherit system;
+          overlays = [
+            fenix.overlays.default
+          ];
+        };
+
+        # Parse the local Cargo.toml so we track the usual rust workflow
+        cargo_toml = builtins.fromTOML (builtins.readFile ./Cargo.toml);
```

Comment:
oh lord is this a standard convention for rust nix flakes (works fine for Oranda because it's a simple workspace but just reading the values out of one Cargo.toml is not in general correct for workspaces...)